lib/ansible/modules/monitoring/zabbix/zabbix_user_group.py

```diff
@@ -0,0 +1,373 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, OVH SAS
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: zabbix_user_group
+short_description: Create/update/delete/dump Zabbix user group
+description:
+    - Create/update/delete/dump Zabbix user group.
+version_added: "2.9"
```

Comment:
```suggestion
version_added: "2.10"
```

The current dev version is 2.10, so I think `version_added` is `2.10`.
https://github.com/ansible/ansible/blob/devel/lib/ansible/release.py

Reply:
Updated :)

Reply:
Thanks for the quick fix :)